app/views/homeworks/edit.html.erb

```diff
@@ -1,6 +1,8 @@
 <h1>Editing Homework</h1>
```

Comment:
can we also update the header while at it? lets use H2's and center it. see the mentor view for example

Reply:
Fixed!